addon/-private/validated-changeset.ts

```diff
@@ -851,14 +857,32 @@ export class BufferedChangeset implements IChangeset {
       }
 
       if (result && isObject(result)) {
-        return normalizeObject(result);
-      }
+        result = normalizeObject(result);
+         let content: Content = this[CONTENT];
```

Comment:
minor spacing issues when you have a chance (should be caught by a linter and is fixed in the `validated-changeset` repo)

Reply:
Yeah, I had to disable all prettier and formatting in my editor to be able to work in these files, otherwise, my changes would be a lot of formatting. 

Anyway, I pushed a fix for the spacing.